### DISCUSSION TITLE
iconify icons in navbar `tools` tag

### DISCUSSION THREAD

**Discussion by ehudkr** [quarto-dev/quarto-cli#8001] [Feature Requests]
### Description

I love the `navbar: tools:` replacing the `navbar: right:`.
However, it currently supports only bootstrap icons (via `icon:`). 
I wonder if it could be possible to support general icons (like iconify via `text:`) -- similar to how the right navigation bar supports them -- to allow adding non-bootstrap icons. 
Thanks

**Comment by mcanouil** [comment]
See:
- https://github.com/quarto-dev/quarto-cli/issues/6359
